BREAKINGCHANGES.md

```diff
@@ -349,6 +364,15 @@ If you prefer platform support vote for the https://experience.dynamics.com/idea
 
 **Solution**: Event has been moved to `codeunit 1440 "RC Headlines Page Common"`, function `OnIsAnyExtensionHeadlineVisible`.
 
+**Error**: _'Codeunit "Type Helper"' does not contain a definition for 'RegexReplace'_\
```

Comment:
Follow the other solution styles.
What is the specific function in the codeunit they should use?